tests/Purl/Test/UrlTest.php

```diff
@@ -307,6 +307,28 @@ public function testFromCurrentServerVariables() {
         $url = Url::fromCurrent();
         $this->assertEquals('http://user:passwd123@jwage.com/', (string) $url);
     }
+    
+    public function testRelativeUrl() 
+    {
+        // test all resource parts
+        $url = new Url("/path1/path2?x=1&y=2#frag");
+        $this->assertFalse($url->isAbsolute());        
+        $this->assertEquals("/path1/path2?x=1&y=2#frag",(string)$url);
```

Comment:
Add a space after the comma and use single quotes for strings:

    $this->assertEquals('/path1/path2?x=1&y=2#frag', (string)$url);